shared/src/ledger/eth_bridge/bridge_pool_vp.rs

```diff
@@ -58,8 +58,18 @@ where
     /// associated with an address
     fn account_balance_delta(&self, address: &Address) -> Option<SignedAmount> {
         let account_key = balance_key(&xan(), address);
-        let before: Amount = self.ctx.read_pre_value(&account_key)?;
-        let after: Amount = self.ctx.read_post_value(&account_key)?;
+        let before: Amount = (&self.ctx)
+            .read_pre_value(&account_key)
+            .unwrap_or_else(|error| {
+                tracing::warn!(?error, %account_key, "reading pre value");
+                None
+            })?;
+        let after: Amount = (&self.ctx)
+            .read_post_value(&account_key)
+            .unwrap_or_else(|error| {
+                tracing::warn!(?error, %account_key, "reading post value");
+                None
+            })?;
```

Comment:
This behaves the same way as before except it will log the error that happened in addition to returning `None`

Reply:
Looks good!